force-app/main/default/classes/SOQL.cls

```diff
@@ -163,9 +195,16 @@ public with sharing class SOQL {
      */
     public static Decimal getTotalRevenueByStage(String stage) {
         // Use AggregateQuery to get the total revenue
-
+        AggregateResult[] rev = 
+        [
+            SELECT SUM(Amount) oppSum 
+            FROM Opportunity 
+            WHERE StageName = :stage OR Amount > 10000
+        ];
+
+        Decimal totalRev = (Decimal) rev[0].get('oppSum'); 
```

Comment:
In a real scenario, we may want to check that our aggregateResults list is not empty before we reference index 0.